### PR TITLE
ARTEMIS-5131 - Add a button to copy messages

### PR DESCRIPTION
**Pull request by andytaylor**
This will be disabled if the management method does not exist

### CODE REVIEW

File: artemis-console-extension/artemis-extension/src/artemis-extension/artemis/artemis-service.ts [hunk 1]
```diff
@@ -589,6 +595,20 @@ class ArtemisService {
...
+
+    doesMethodExist = (broker: MBeanNode | undefined, queue: string, method: string): boolean => {
+        if(broker) {
+            var queueMBean = broker.parent?.find(node => { 
+                return node.propertyList?.get('subcomponent') === 'queues' && node.name === queue 
+            })
+            return queueMBean?queueMBean.hasOperations(method): false;
+        }
+        return false;
+    }
```
Comment (lavocatt) [inline artemis-console-extension/artemis-extension/src/artemis-extension/artemis/artemis-service.ts:611]: Could there be a situation where there are several `queueMBean` validating the condition in the find? If so, I would change the find to a filter and try to find the operation in at least one of them.
If not, ignore this

Comment (andytaylor) [reply artemis-console-extension/artemis-extension/src/artemis-extension/artemis/artemis-service.ts:611]: nope that cant happen as its a 1 to 1 with address name and queue name

File: artemis-console-extension/artemis-extension/src/artemis-extension/artemis/messages/MessagesTable.tsx [hunk 2]
```diff
@@ -425,7 +497,7 @@ export const MessagesTable: React.FunctionComponent<MessageProps> = props => {
           <Button key="save" variant="primary" onClick={onSave}>
             Save
           </Button>,
-          <Button key="close" variant="secondary" onClick={handleColumnsModalToggle}>
+          <Button key="cloButtonPropsse" variant="secondary" onClick={handleColumnsModalToggle}>
```
Comment (brusdev) [inline artemis-console-extension/artemis-extension/src/artemis-extension/artemis/messages/MessagesTable.tsx:500]: is `cloButtonPropsse` a typo?

Comment (andytaylor) [reply artemis-console-extension/artemis-extension/src/artemis-extension/artemis/messages/MessagesTable.tsx]: yes, I have fixed

**Pull request merged**